posthog/tasks/periodic_digest.py

```diff
@@ -68,8 +81,22 @@ def get_teams_with_new_event_definitions(end: datetime, begin: datetime) -> Quer
     return EventDefinition.objects.filter(created_at__gt=begin, created_at__lte=end).values("team_id", "name", "id")
 
 
-def get_teams_with_new_playlists(end: datetime, begin: datetime) -> QuerySet:
-    return (
+def get_teams_with_new_playlists(end: datetime, begin: datetime) -> list[CountedPlaylist]:
```

Comment:
do we want this on new playlists only? I imagine we'd want this on playlists that someone made weeks ago, like "signup interactions" they won't make every week but is still valid every week to know they have new stuff there

Reply:
fair... makes me think it would be great to sort these so that we're highlighting the ones with the newest recordings as well as the most 🤔

Reply:
Does number of recordings give a lot of signal? Ie I might have a playlist that gets 10 recordings per week but it's for "customers with failed subscription attempts" and I really care about those... 

I'd probably sort by "most recently viewed" playlists. Or view frequency? IDK if we have metrics on that though. For now we can just do whatever metric we have and adjust from there.

We can have two categories here if we want - new playlists + count, interesting playlists + count